src/material-experimental/mdc-menu/menu-item.html

```diff
@@ -1,4 +1,4 @@
-<ng-content></ng-content>
+<span class="mdc-list-item__primary-text"><ng-content></ng-content></span>
```

Comment:
For the MDC list update I'm working on, I was wrapping `__primary-text` et all in [`mdc-list-item__content`](https://cs.opensource.google/material-components/material-components-web/+/master:packages/mdc-list/_evolution-mixins.scss;l=135?q=%22mdc-list-item__content%22&ss=material-components%2Fmaterial-components-web). Should we do that here too? I assume it currently does not matter visually, but it might change in the future and conceptually I think MDC defines three sections in a list item. i.e. `__start`, `__content` and `__end`.

Reply:
I didn't do it, because the content class comes with `overflow-ellipsis` which will truncate the text. One of the complaints we've had about `mat-select` and `mat-menu` is that we don't allow multi-line text and I wanted to support that for the MDC version.

Reply:
I see. That makes sense. I wish we could capture that somewhere, but at least it's now in the PR 😄